serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

```diff
@@ -0,0 +1,245 @@
+      end: true
+----
+
+<1> The annotation `sw.kogito.kie.org/profile: dev` tells the operator to deploy your workflow using the devlopment profile. This means that the operator will build a running instance of the workflow ready to receive changes during your development cycle.
+
+<2> In the `spec` attribute goes the workflow definition as described by the xref:getting-started/cncf-serverless-workflow-specification-support.adoc[CNCF Serverless Workflow specification]. So if you already have a workflow definition, you can use it there. Alternatively, you can use the xref:tooling/serverless-workflow-editor/swf-editor-overview.adoc[editors to create your workflow definition].
+
+== Deploying a New Workflow Service
+
+.Prerequisites
+* You have xref:cloud/operator/install-serverless-operator.adoc[installed the {operator_name}]
+* You have created a new {product_name} Kubernetes YAML file
+
+Having a new Kubernetes workflow definition in a YAML file (you can use the above example), you can deploy it in your cluster with the following command:
+
+.Deploying a new KogitoServerlessWorkflow custom resource in Kubernetes
+[source,bash,subs="attributes+"]
+----
+kubectl apply -f <your_file> -n <your_namespace>
+----
+
+Alternatively, you can try one of the examples available in the operator repository:
+
+.Deploying the greeting workflow example
+[source,bash,subs="attributes+"]
+----
+kubectl apply -f https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow_devmode.yaml -n <your_namespace>
+----
+
+[TIP]
+====
+Replace `<your_namespace>` with the namespace you're using to deploy your applications
+====
+
+You can follow the workflow status to check if everything is fine with:
+
+.Checking the workflow status
+[source,bash,subs="attributes+"]
+----
+kubectl get workflow -w
+----
+
+You should see the workflow conditions evolving to `READY` in a few seconds:
+
+.Example workflow deployment status
+[source,bash,subs="attributes+"]
+----
+NAME       PROFILE   VERSION   ADDRESS   READY   REASON
+greeting   dev       0.0.1               False   WaitingForDeployment
+greeting   dev       0.0.1               True
+----
+
+[TIP]
+====
+The `REASON` field gives you a cue about the current workflow status.
+====
+
+You can make changes to the workflow YAML using any Kubernetes editor such as `kubectl`. The operator ensures that the latest workflow definition is running and ready. 
+This way, you can include the workflow application in your development scenario and start making requests to it.
+
+== Check the workflow application is running
+
+In order to check that the {product_name} Greeting application is up and running, you can try to perform a test HTTP call. First, you must expose the service:
+
+.Exposing the workflow application
+[source,bash,subs="attributes+"]
+----
+kubectl patch svc greeting -p '{"spec": {"type": "NodePort"}}'
```

Comment:
isn't the operator doing this already?

Reply:
Nops: https://github.com/kiegroup/kogito-serverless-operator/pull/78